picamera2/encoders/h264_encoder.py

```diff
@@ -60,6 +61,16 @@ def _start(self):
         fmt.fmt.pix_mp.plane_fmt[0].sizeimage = 512 << 10
         fcntl.ioctl(self.vd, VIDIOC_S_FMT, fmt)
 
+        ext = v4l2_ext_controls()
+        extc = v4l2_ext_control()
+        extc.id = (0x00990000 | 0x900) + 226  # V4L2_CID_MPEG_VIDEO_REPEAT_SEQ_HEADER
```

Comment:
That long V4L2_CID thing probably wants to go into our v4l2.py, right?

Also, I think "repeat seq header" should be an optional parameter when the encoder object is created (along with the bitrate). We should probably also have the GOP size there too.